addons/web/static/src/core/model_field_selector/model_field_selector_popover.js

```diff
@@ -15,6 +16,7 @@ export class ModelFieldSelectorPopover extends Component {
         this.fieldKeys = [];
         this.searchValue = "";
         this.fullFieldName = this.fieldNameChain.join(".");
+        !this.env.isSmall && useAutofocus();
```

Comment:
```suggestion
        if (!this.env.isSmall) {
            useAutofocus();
        }
```

Reply:
is there an existing test you can simply add an assertion that the autofocus is done ?